frontend/package.json

```diff
@@ -6,13 +6,15 @@
   "scripts": {
     "build": "remix vite:build",
     "dev": "remix vite:dev --port 5101",
-    "preview": "wrangler pages dev ./build/client --port=5100 --binding SCRIPT_FRONTEND_URL=http://localhost:5100/ --compatibility-date=2024-09-23 --compatibility-flag=nodejs_compat"
+    "preview": "pnpm run build && wrangler pages dev ./build/client",
+    "deploy": "pnpm run build && wrangler pages deploy"
   },
   "dependencies": {
     "@headlessui/react": "^1.7.19",
     "@interledger/open-payments": "^6.13.2",
     "@paralleldrive/cuid2": "^2.2.2",
     "@remix-run/cloudflare": "^2.16.5",
```

Comment:
Let's update this package as well, keeping in line with `@remix-run/cloudflare-pages`

Reply:
Along with other remix packages?